app/views/claims/teacher_reference_number.html.erb

```diff
@@ -8,11 +8,11 @@
           <%= form.label :teacher_reference_number, t("tslr.questions.teacher_reference_number"), class: "govuk-label govuk-label--xl" %>
         </h1>
 
-        <span class="govuk-hint">This is on the certificate you got when you qualified as a teacher, or your school can tell you.</span>
+        <span class="govuk-hint">You can get this from your school, the certificate you got when you qualified as a teacher or from the <a href="https://www.gov.uk/guidance/individual-teacher-records-information-for-teachers#contact">teacher qualifications helpdesk</a>.</span>
```

Comment:
Should this be a translation?

Reply:
Perhaps we should so we can edit questions and hints in the same location.